store/views.py

```diff
@@ -77,6 +99,42 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    post_data = request.POST
+    try:
+        bookCopy = BookCopy.objects.get( id__exact=post_data['bid'], borrower=request.user )
+        message = 'success'
+        bookCopy.status = True
+        bookCopy.borrower = None
+        bookCopy.save()
+    except:
```

Comment:
There must be a validation in the backend when a user is returning the book, to make sure that he has only borrowed the book. Otherwise, a simple POST request will make the BookCopy to be returned, and would set its status as True.